test/integration/patients/patient/sidebar.js

```diff
@@ -821,6 +821,120 @@ context('patient sidebar', function() {
       .should('not.contain', 'Workspace Two');
   });
 
+  specify('workspace specific widgets setting', function() {
+    const dob = testDateSubtract(10, 'years');
+
+    const fields = {
+      'test-field': {
+        id: '1',
+        name: 'test-field',
+        value: '1',
+      },
+    };
+
+    cy
+      .routesForPatientDashboard()
+      .routeFormDefinition()
```

Comment:
I don't think any of these forms routes are in use

Reply:
I think you can get away with just  `routesForPatientDashboard` and `routeWorkspaces`